setup.py

```diff
@@ -43,7 +43,7 @@
     packages=find_packages(),
     install_requires=[
         'Fabric3',
-        'fauxfactory==2.0.9',
+        'fauxfactory==3.0.2',
```

Comment:
robottelo 6.3.z branch needs fauxfactory==2.1.1 ?

Reply:
This Pr is for 6.4.z branch.